lib/SPIRV/libSPIRV/SPIRVEntry.cpp

```diff
@@ -344,7 +344,20 @@ std::string SPIRVEntry::getDecorationStringLiteral(Decoration Kind) const {
   for (SPIRVWord I = 0; I < Loc->second->getLiteralCount(); ++I)
     Literals.push_back(Loc->second->getLiteral(I));
 
-  return getString(Literals.cbegin(), Literals.cend());
+  return getString(Literals);
+}
+
+std::vector<std::string>
+SPIRVEntry::getDecorationStringLiterals(Decoration Kind) const {
```

Comment:
This function looks absolutely the same as the previous one. I've spent several minutes to find differences and spot these two additional `s` symbols across ~8 lines of code.

I've searched for usages of `getDecorationStringLiteral` and looks like it is used only 3 times. Would it be better to completely replace `getDecorationStringLiteral` by the new `getDecorationStringLiterals`?

UPD: looks like I've found the reason for duplication: new `getDecorationStringLiterals` has to use new `getStrings` method under the hood, while the original `getString` method (used by `getDecorationStringLiteral`) is heavily used in `LLVMToSPIRVDbgTran.cpp` and some other places. So, it will be a huge change. Not asking then to refactor this right now - will leave this comment for everyone who is also wondering why is this code looks pretty the same as the existing.

Reply:
> while the original getString method (used by getDecorationStringLiteral) is heavily used in LLVMToSPIRVDbgTran.cpp and some other places

In LLVMToSPIRVDbgTran.cpp we heavily use `SPIRVModuleImpl::getString()` method not a free function `getString()` from SPIRVUtil.h

Reply:
Then, it is probably better to update existing `getDecorationStringLiteral` function (or replace it with the new one, more generic) instead of creating an almost full copy.